tests/components/gdacs/test_geo_location.py

```diff
@@ -97,6 +98,8 @@
         all_states = hass.states.async_all()
         # 3 geolocation and 1 sensor entities
         assert len(all_states) == 4
+        entity_registry = await async_get_registry(hass)
+        assert len(entity_registry.entities) == 4
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5253267)